src/add/operator/buffer.ts

```diff
@@ -4,4 +4,8 @@ import {buffer} from '../../operator/buffer';
 
 Observable.prototype.buffer = buffer;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  interface Observable<T> {
+    buffer: (closingNotifier: Observable<any>) => Observable<T[]>;
```

Comment:
I was planning to add interface exports to the operator files, so that the signature would be:

```
declare module '../../Observable' {
  interface Observable<T> {
    buffer: buffer<T>;
  }
}
```

The problem with doing just `typeof buffer` I think will be that the output type for consumers will be `Observable<{}>` instead of `Observable<T>`.